Utilities/DataLoading/EdFi.LoadTools/ApiClient/DependenciesRetriever.cs

```diff
@@ -106,16 +106,20 @@ private async Task LoadDependenciesAsync()
 
         private async Task<string> LoadJsonStringAsync(string dependenciesUrl)
         {
-            using (var client = new HttpClient
+            HttpClientHandler handler = new HttpClientHandler
             {
-                Timeout = new TimeSpan(0, 0, 5, 0)
-            })
+                ServerCertificateCustomValidationCallback = (sender, cert, chain, sslPolicyErrors) => true
```

Comment:
Why are we sneaking a certification validation ignore in on a replacement for the container package?

Reply:
Ohh I was testing with docker, will create a separate ticket, and remove this change

Reply:
ODS-4720